app/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -0,0 +1,127 @@
+    <string name="last_seen">Visto último</string>
+    <string name="conn_status_closed">Fechado</string>
+    <string name="copy_to_clipboard">Copiar ao Clipboard</string>
+    <string name="copied_to_clipboard">Copiado ao Clipboard!</string>
+    <string name="connections_csv_fields_v1">IPProto,SrcIP,SrcPort,DstIp,DstPort,Uid,App,Proto,Status,Info,BytesSent,BytesRcvd,PktsSent,PktsRcvd,FirstSeen,LastSeen</string>
+    <string name="save_to_file">Salvar em arquivo</string>
+    <string name="file_saved">Arquivo salvo com sucesso</string>
+    <string name="older_connections_notice">%1$d conexões mais antigas não mostradas</string>
+    <string name="about_text">PCAPdroid é uma ferramenta de captura e monitoramento de rede de software livre que funciona sem acesso Root.</string>
+    <string name="app_no_warranty">PCAPdroid é distribuído na esperança de ser útil, mas SEM QUALQUER TIPO DE GARANTIA; nem mesmo a garantia implícita de COMERCIALIZAÇÃO ou ADEQUAÇÃO PARA UM PROPÓSITO PARTICULAR. Veja a Licença Pública Geral GNU para mais detalhes.</string>
+    <string name="gpl_license_link">Licença GPLv3</string>
+    <string name="get_app">Obtenha-o:</string>
+    <string name="tls_how_to">Como configurar a decriptação de TLS</string>
+    <string name="unknown_app">Desconhecido</string>
+    <string name="vpn_prefs">VPN</string>
+    <string name="enable_ipv6">Ativar IPv6</string>
+    <string name="enable_ipv6_summary">Permitir conexões de IPv6. Só deve ser usado em redes com um Gateway ao IPv6.</string>
+    <string name="error">Erro</string>
+    <string name="conn_status_unreachable">Inacessível</string>
+    <string name="other_prefs">Outro</string>
+    <string name="app_language">Idioma do App</string>
+    <string name="lang_english">Inglês</string>
+    <string name="system_default">Padrão do sistema</string>
+    <string name="no_activity_file_selection">Nenhum app de seleção de arquivos foi encontrado</string>
+    <string name="file_saved_with_name">Arquivo %1$s salvo com sucesso</string>
+    <string name="theme_light">Claro</string>
+    <string name="theme_dark">Escuro</string>
+    <string name="app_theme">Tema</string>
+    <string name="enable_socks5_proxy">Ativar Proxy de SOCKS5</string>
+    <string name="enable_socks5_proxy_summary">Quando ativado, todas as conexões de TCP serão redirecionadas ao proxy de SOCKS5 especificado.</string>
+    <string name="proxy">Proxy</string>
+    <string name="proxy_ip_address">Endereço IP de Proxy</string>
+    <string name="proxy_port">Porta de Proxy</string>
+    <string name="root_capture">Capturar via Root</string>
+    <string name="root_capture_summary">A captura de pacotes via Root permite que o PCAPdroid funcione simultaneamente com outros apps de VPN.</string>
+    <string name="donate">Doar</string>
+    <string name="http_request">Solicitação HTTP</string>
+    <string name="root_log">Root Log</string>
+    <string name="request_plaintext">Solicitar texto simples</string>
```

Comment:
Here `request` is a noun, not a verb, it's like "the plaintext of the request"

Reply:
good catch!

Reply:
`request_plaintext` was the one to fix, I'll fix it after merge

Reply:
See https://github.com/emanuele-f/PCAPdroid/commit/0c3f5a16d55a2fd525c43c3a2f7a16af6102956e